src/screens/SendToken/SendTokenTransaction.js

```diff
@@ -92,29 +95,30 @@ const getTransactionSuccessTitle = (props) => {
   return t('transactions.title.transactionSuccess');
 };
 
-const CancelText = styled(MediumText)`
-  color: ${themedColors.negative};
-  font-size: ${fontSizes.medium}px;
-`;
+function SendTokenTransaction() {
+  const dispatch = useDispatch();
+  const navigation = useNavigation();
+  const fromAddress = useRootSelector(activeAccountAddressSelector);
 
-const ButtonWrapper = styled.View`
-  width: 100%;
-  margin: 0px ${spacing.layoutSides}px 20px;
-`;
+  const isSuccess: boolean = useNavigationParam('isSuccess');
+  const error: string = useNavigationParam('error', '');
```

Comment:
error might not be present at all, maybe we want to keep this way and check accordingly? wdyt?

Reply:
Consider not defaulting to empty string
```suggestion
  const error: ?string = useNavigationParam('error');
```